doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/5_1_0.xhtml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<link rel="stylesheet" type="text/css" href="new.css" />
+<title>What's new in hale»studio 5.0.1?</title>
+</head>
+<body>
+	<h1>What's new in hale»studio 5.1.0?</h1>
+
+	<ul>
+		<li>Added an option to overwrite content in Geopackage Writer instead of appending content</li>
+		<li>Added a parameter to the GeoPackage writer that allows creating tables for all mapping-relevant target types</li>
+	</ul>
```

Comment:
This list should reflect the most important points from the changelog and include the points that are shown in more detail further down.